tests/MongoDB.Driver.Legacy.Tests/MongoCollectionTests.cs

```diff
@@ -2414,18 +2349,9 @@ public void TestInsertBatchMultipleBatchesWriteConcernEnabledContinueOnErrorFals
             CheckExpectedResult(expectedResult, result);
 
             var results = ((IEnumerable<WriteConcernResult>)exception.Data["results"]).ToArray();
-            if (_primary.Supports(FeatureId.WriteCommands))
-            {
-                // it the opcode was emulated there will just be one synthesized result
-                Assert.Equal(1, results.Length);
-                Assert.Equal(true, results[0].HasLastErrorMessage);
-            }
-            else
-            {
-                Assert.Equal(2, results.Length);
-                Assert.Equal(false, results[0].HasLastErrorMessage);
-                Assert.Equal(true, results[1].HasLastErrorMessage);
-            }
+            // it the opcode was emulated there will just be one synthesized result
```

Comment:
Let's fix minor typo (that was already there...):

    // because the opcode was emulated there will just be one synthesized result

Reply:
Done.